backend/utils.js

```diff
@@ -14,3 +14,24 @@ export const generateToken = (user) => {
     }
   );
 };
+
+export const isAuth = (req, res, next) => {
+  const authorization = req.headers.authorization;
+  if (authorization) {
+    const token = authorization.slice(7, authorization.length); // Bearer XXXXXX
```

Comment:
xhr.js:210          POST http://localhost:3001/api/orders 401 (Unauthorized)

Reply:
Hello! Did you fix this one?

Reply:
I am facing the same error. Did anyone solve this error
POST http://localhost:3000/api/orders 401 (Unauthorized)